app/src/flavorGplay/res/values/string-not_translatable.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_donate_url" translatable="false"><![CDATA[https://gsantner.net/project/markor.html?project=markor&source=inapp_supdon_gplay]]></string>
```

Comment:
Not allowed to link to donation page from within google play app. don't want to get the app banned for this.

Reply:
hmmk